content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -7,7 +7,7 @@ With DVC experiments and checkpoints, you can:
 
 - Implement the best practice in deep learning to save your model weights as
   checkpoints.
-- Track all code and data changes corresponded to the checkpoints.
+- Track all code and data changes corresponded to a checkpoint.
```

Comment:
"The" was OK (refers to "as checkpoints" in the bullet above. But I think it should be "corresponding"

Reply:
I think we don't refer to a specific checkpoint, _the_ makes me to think _there is a single checkpoint for which we can associate all the code and data._ You're right it should be _corresponding_.